pyproject.toml

```diff
@@ -52,3 +44,14 @@ disable = """
     raise-missing-from,
     too-many-instance-attributes,
 """
+
+[tool.ruff]
+line-length = 79
```

Comment:
79? I usually see 88 instead

Reply:
For the API development we set [PEP-8](https://peps.python.org/pep-0008/#maximum-line-length) as a north